docs/src/prototypes/chatMessageWithPopover/Popover.tsx

```diff
@@ -36,19 +36,33 @@ class Popover extends React.Component<PopoverProps, PopoverState> {
     opacity: 0,
 
     '& .smile-emoji': {
-      display: 'none',
+      position: 'absolute',
+      opacity: 0,
+      zIndex: -1,
     },
 
     '&.focused .smile-emoji': {
-      display: 'flex',
+      position: 'initial',
+      zIndex: 'initial',
+      opacity: 1,
     },
 
     '&:hover .smile-emoji': {
-      display: 'flex',
+      position: 'initial',
+      zIndex: 'initial',
+      opacity: 1,
     },
   })
 
   render() {
+    const overridenToolbarBehavior: Accessibility = (props: any) => {
```

Comment:
there is no need to create this function on each `render` - this may be put out of the `render` scope